ajax/libs/pulltorefresh.js/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "pulltorefresh.js",
+  "filename": "dist/pulltorefresh.min.js",
+  "description": "Pull To Refresh",
```

Comment:
Lets use this one description: "A quick and powerful plugin for your pull-to-refresh needs in your webapp."